<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>190</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Noise Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSpinBox" name="SpnOctaves">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Details of noise</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Octaves:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Lacunarity:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ChkStrAbvZero">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Generate noise above zero</string>
     </property>
     <property name="text">
      <string>Generate noise above zero</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnAssign">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Assign</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Attribute:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CmbAttr">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>20</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Selected object attributes</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpnLacunarity">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Increase in details in each octave</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnRefreshAttr">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Refresh selected object attributes</string>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpnFrequency">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The speed of noise</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
     <property name="value">
      <double>4.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gain:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpnGain">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Noise strength</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Strength:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpnStrength">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Overall strength of noise</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnRemove">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>CmbAttr</tabstop>
  <tabstop>BtnRefreshAttr</tabstop>
  <tabstop>SpnFrequency</tabstop>
  <tabstop>SpnOctaves</tabstop>
  <tabstop>SpnStrength</tabstop>
  <tabstop>SpnLacunarity</tabstop>
  <tabstop>SpnGain</tabstop>
  <tabstop>ChkStrAbvZero</tabstop>
  <tabstop>BtnAssign</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
